source/draw/Color.ooc

```diff
@@ -70,6 +71,7 @@ ColorUv: cover {
 		this v = color v
 	}
 	init: func@ (=u, =v)
+	// TODO: use valid default colors
```

Comment:
Yes, this can preserve correct hue on oversaturated images so that the receiving television can scale it down.